src/libmusig2.h

```diff
@@ -27,12 +27,15 @@
  *              : state: The state of musig2.
  *              : nr_signers: The number of signers.
  * */
+ // todo: this should contain the 'signing' data, so maybe combined with the params.
 typedef struct{
     secp256k1_context* ctx;
     secp256k1_pubkey aggr_pk;
     secp256k1_pubkey aggr_R;
```

Comment:
Doesn't seem like we need it. We could remove this, and initialise this context in a `precomputation` phase for the signer.

Reply:
We will use `aggr_R` from the struct `musig2_partial_signature` in the partial signature step.